docs/compendium/substitute.md

```diff
@@ -25,7 +36,90 @@ introduction...
 ### Error responses
 
 ```json
+401 Unauthorized
+
+{"error":"not authenticated or not allowed"}
+```
+
+```json
+404 Not Found
+
+{"error":"compendia ids not found"}
```

Comment:
In the text please distinguish "base compendium not found" vs. "overlay compendium not found". A more specific message helps client developers and is easily done in the backend.